### ISSUE TITLE
License of JellyFish samples

### ISSUE THREAD

**Issue by dabrain34** [jellyfin/jellyfin.org#1216]
On this page https://repo.jellyfin.org/jellyfish/, there are samples we might use for a project, could you please share with us the license of those files.

We might use those files in RAW YUV format.

**Comment by dabrain34** [comment]
Thanks for your answer. From my initial research It was also not very clear if we can replicate these samples without any credits in our project. 
I can see that https://kodi.wiki/view/Samples and https://matts-soup.blogspot.com/2016/05/test-videos-4k-uhd-bitrate-tests-sample.html mention @ScottAllyn01 which might be the initial author I guess. I dont know if it would be enough ..

**Comment by joshuaboniface** [comment]
I did receive an indirect response, which amounted to confirmation that the original videos were proprietary but given to ScottAllyn01 as a perpetual open license, so technically these should be usable by 3rd parties with his permission. That said I did not explicitly receive that permission.

At the same time, @gnattu on the team also started work on his own set of test videos, not only to solve the licensing issue, but also to better support the wider range of codecs that Jellyfin now supports, and thus provide a wider test suite. Those videos are now live on the repo: https://repo.jellyfin.org/test-videos. They're CC BY-SA licensed and thus should be usable inside any FLOSS project without issue.

We've also deprecated the Jellyfish video set by moving them into our repo archive which is not distributed to worldwide mirrors. So while that video set will remain hosted by us indefinitely for posterity (or until asked not to), downloading them will be relatively slow.

I think that resolves this one unless I hear more.
